fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/responses/RdfStreamStreamingOutput.java

```diff
@@ -64,6 +64,8 @@
 
     private final RDFFormat format;
 
+    private final MediaType mediaType;
```

Comment:
We may need a brief comment on these fields explaining how `format` and `mediaType` work together to develop a specific choice of serialization.